aztk/utils/helpers.py

```diff
@@ -26,6 +27,11 @@ def get_cluster(cluster_id, batch_client):
 
     return aztk.models.Cluster(pool, nodes)
 
+def validate_docker_repo(docker_repo):
+    repository, tag = docker_repo.split(":")
+    response = requests.get("https://index.docker.io/v1/repositories/{0}/tags/{1}".format(repository, tag))
```

Comment:
This won't work for private docker repositories (e.g. Azure Container Repository - ACR).

The repository can be in the form:
private.repo.io/imagename:tag

We need to be able to handle any version of this. Probably worth looking into whether or not there is a docker API to check this.